src/components/SummaryBox/SummaryBox/SummaryBox.test.tsx

```diff
@@ -47,34 +47,32 @@ const testSummaryBoxContent = (
 const customProps = {
   role: 'complementary',
   className: 'custom-class-name',
-  heading: 'Example heading',
 }
 
 describe('SummaryBox component', () => {
   it('renders without errors', () => {
-    const { getByRole } = render(
-      <SummaryBox heading="Key information">{testSummaryBoxContent}</SummaryBox>
+    const { getByTestId } = render(
+      <SummaryBox>{testSummaryBoxContent}</SummaryBox>
     )
 
-    expect(getByRole('heading')).toBeInTheDocument()
+    expect(getByTestId('summary-box')).toBeInTheDocument()
```

Comment:
This test _was_ (although the name didn't reflect it explicitly) the only test that validated that a heading as being rendered. Perhaps `testSummaryBoxContent` should be updated to include the heading now?

Reply:
I don't think that's needed: now the heading is just... part of the children of SummaryBox.

Reply:
Right, but shouldn't the tests somewhat reflect how we expect the component to be used? In which case, the `render` should include those tags with `testSummaryBoxContent` rendered inside the `SummaryBoxContent`.

And then I think there's still value in checking for the header to be rendered within. What we could probably do here is combine the second test (renders passed in children) with the first test, make sure we do a check for the header, and call it a day